app/src/main/java/ca/ubc/best/mint/museandroidapp/vm/FlankerStimulus.java

```diff
@@ -45,4 +107,13 @@ public String asText() {
     Log.d("MINT", "Stimulus array: " + stimulusArray.toString());
     return stimulusArray;
   }
+*/
+
+  private static final List<FlankerStimulus> VALUES = Collections.unmodifiableList(Arrays.asList(values()));
+  private static final int SIZE = VALUES.size();
+  private static final Random RANDOM = new Random();
+
+  public static FlankerStimulus randomStimulusCue()  {
+    return VALUES.get(RANDOM.nextInt(SIZE));
```

Comment:
Is this still being used?

I think you have Collections.shuffle(stimulusCueArray) below, is there a need to still get a random cue anywhere?